requirements.txt

```diff
@@ -0,0 +1,170 @@
+# This file may be used to create an environment using:
+# $ conda create --name <env> --file <this file>
+# platform: linux-64
```

Comment:
Can you tell me how to generate this?
let's have requirements.txt for ubuntu 18(your pc) and ubuntu 20 (mine)

Reply:
e.g. installing boost 1.76 failed in my PC.
Need to specify version conda install boost==1.71 to make this work

Reply:
For me it worked.